django_admin_shell/templates/django_admin_shell/shell.html

```diff
@@ -24,6 +24,14 @@ <h5> Django version : {{django_version}} </h5>
         </div>
     </div>
 
+    <script>
+        function save_snippet(code) {
+            var encoded = encodeURIComponent(code);
+            var url = document.location.origin + "/admin/admin_shell/savedsnippets/add/?code=" + encoded;
```

Comment:
In here it would be better to use a django tag to get a url to proper site: 
Please look here: https://docs.djangoproject.com/en/5.1/ref/templates/builtins/#url
And here : https://docs.djangoproject.com/en/5.1/ref/contrib/admin/#reversing-admin-urls
it should be 
```
var url = document.location.origin + "{% url 'admin:admin_shell_savedsnippets_add' %}?code=" + encoded;
```